src/arch/sparc64/CodeGen.zig

```diff
@@ -1160,6 +1161,21 @@ fn airBr(self: *Self, inst: Air.Inst.Index) !void {
     return self.finishAir(inst, .dead, .{ branch.operand, .none, .none });
 }
 
+fn airTrap(self: *Self) !void {
+    // ta 0x05
+    _ = try self.addInst(.{
+        .tag = .tcc,
+        .data = .{
+            .trap = .{
+                .is_imm = true,
+                .cond = .al,
+                .rs2_or_imm = .{ .imm = 0x05 },
+            },
+        },
+    });
+    return self.finishAirBookkeeping();
+}
+
 fn airBreakpoint(self: *Self) !void {
     // ta 0x01
     _ = try self.addInst(.{
```

Comment:
Reference for these two: https://github.com/llvm/llvm-project/blob/1df8de238b5af1caca0c6da8a106c9453d84be6a/llvm/lib/Target/Sparc/SparcInstrInfo.td#L1100-L1104